api/tests/unit/domain/services/certification/certification-result-service_test.js

```diff
@@ -1225,4 +1226,75 @@ describe('Unit | Service | Certification Result Service', function() {
 
   });
 
+  describe('#computeAnswersSuccessRate', () => {
+
+    context('when no answers is given', () => {
+
+      it('should have a trust level has unknown', () => {
+        // given
+        const answers = [];
+
+        // when
+        const successRate = certificationResultService._computeAnswersSuccessRate(answers);
```

Comment:
Comme noté dans /certification-result-service.js, la présence de l'underscore fait penser à tord à une fonction privée, donc non testable, alors qu'elle est en fait publique